### PR TITLE
Add User/Team tab to AWX projects

### PR DESCRIPTION
**Pull request by ZitaNemeckova**

### CODE REVIEW

**Comment by vishsanghishetty** [comment]
QA'ed locally, works as expected

**Review by vishsanghishetty — APPROVED** [review]

File: frontend/awx/resources/projects/ProjectPage/ProjectPage.tsx [hunk 1]
```diff
@@ -50,6 +50,8 @@ export function ProjectPage() {
       { label: t('Access'), page: AwxRoute.ProjectAccess },
       { label: t('Schedules'), page: AwxRoute.ProjectSchedules },
       { label: t('Job templates'), page: AwxRoute.ProjectJobTemplates },
+      { label: t('Users'), page: AwxRoute.ProjectUsers },
```
Comment (vidyanambiar) [inline frontend/awx/resources/projects/ProjectPage/ProjectPage.tsx:53]: Let's rename the tabs as User Access and Team Access to be consistent with the experience for EDA resources.

File: frontend/awx/resources/projects/ProjectPage/ProjectPage.tsx [hunk 2]
```diff
@@ -50,6 +50,8 @@ export function ProjectPage() {
       { label: t('Access'), page: AwxRoute.ProjectAccess },
```
Comment (vidyanambiar) [inline frontend/awx/resources/projects/ProjectPage/ProjectPage.tsx:50]: Can we get rid of this "Access" tab and the accompanying route/navigation please? This is replaced by the User Access and Team Access tabs.

**Review by vidyanambiar — COMMENTED** [review]
Looks good 👍 

Can we get some component tests added please?

File: frontend/awx/resources/projects/ProjectPage/ProjectPage.tsx (hunk 1 above)
Comment (ZitaNemeckova) [reply frontend/awx/resources/projects/ProjectPage/ProjectPage.tsx:53]: Fixed.

File: frontend/awx/resources/projects/ProjectPage/ProjectPage.tsx (hunk 2 above)
Comment (ZitaNemeckova) [reply frontend/awx/resources/projects/ProjectPage/ProjectPage.tsx:50]: Fixed.

Comment (vidyanambiar) [reply frontend/awx/resources/projects/ProjectPage/ProjectPage.tsx:50]: @ZitaNemeckova  Still seeing the redundant Access tab, is there a pending commit?

**Review by vidyanambiar — APPROVED** [review]

File: frontend/awx/main/routes/useAwxProjectRoutes.tsx [hunk 3]
```diff
@@ -103,12 +101,32 @@ export function useAwxProjectRoutes() {
               path: 'schedules',
               element: <SchedulesList sublistEndpoint={awxAPI`/projects`} />,
             },
+            {
+              id: AwxRoute.ProjectUsers,
+              path: 'users',
```
Comment (keithjgrant) [inline frontend/awx/main/routes/useAwxProjectRoutes.tsx:106]:
```suggestion
              path: 'user-access',
```

File: frontend/awx/main/routes/useAwxProjectRoutes.tsx [hunk 4]
```diff
@@ -103,12 +101,32 @@ export function useAwxProjectRoutes() {
...
+              element: <ProjectUsers />,
+            },
+            {
+              id: AwxRoute.ProjectTeams,
+              path: 'teams',
```
Comment (keithjgrant) [inline frontend/awx/main/routes/useAwxProjectRoutes.tsx:111]:
```suggestion
              path: 'team-access',
```

File: frontend/awx/main/routes/useAwxProjectRoutes.tsx [hunk 5]
```diff
@@ -103,12 +101,32 @@ export function useAwxProjectRoutes() {
...
+            },
             {
               path: '',
               element: <Navigate to="details" />,
             },
           ],
         },
+        {
+          id: AwxRoute.ProjectAddUsers,
+          path: ':id/users/add-users',
```
Comment (keithjgrant) [inline frontend/awx/main/routes/useAwxProjectRoutes.tsx:122]:
```suggestion
          path: ':id/user-access/add',
```

File: frontend/awx/main/routes/useAwxProjectRoutes.tsx [hunk 6]
```diff
@@ -103,12 +101,32 @@ export function useAwxProjectRoutes() {
...
+          element: <AwxProjectAddUsers />,
+        },
+        {
+          id: AwxRoute.ProjectAddTeams,
+          path: ':id/users/add-teams',
```
Comment (keithjgrant) [inline frontend/awx/main/routes/useAwxProjectRoutes.tsx:127]:
```suggestion
          path: ':id/team-access/add',
```

**Review by keithjgrant — CHANGES_REQUESTED** [review]
Looks good! I'm recommending a few URL changes to match a discussion @vidyanambiar and I had about making these sorts of URLs more consistent

**Comment by ZitaNemeckova** [comment]
@keithjgrant Fixed! Thanks for pointing it out.

**Review by keithjgrant — APPROVED** [review]

**Comment by keithjgrant** [comment]
It looks like the Projects test(s) need to be updated to check for the new tab labels

**Pull request merged**